docs/reference/query-languages/esql/esql-search-tutorial.md

```diff
@@ -459,16 +399,244 @@ FROM cooking_blog METADATA _score
 | LIMIT 1000
 ```
 
+## Step 8: Semantic search and hybrid search
+
+### Index semantic content
+
+{{es}} allows you to semantically search for documents based on the meaning of the text, rather than just the presence of specific keywords. This is useful when you want to find documents that are conceptually similar to a given query, even if they don't contain the exact search terms.
+
+ES|QL supports semantic search when your mappings include fields of the [`semantic_text`](/reference/elasticsearch/mapping-reference/semantic-text.md) type. This example mapping update adds a new field called `semantic_description` with the type `semantic_text`:
+
+```console
+PUT /cooking_blog/_mapping
+{
+  "properties": {
+    "semantic_description": {
+      "type": "semantic_text"
+    }
+  }
+}
+```
+
+Next, index a document with content into the new field:
+
+```console
+POST /cooking_blog/_doc
+{
+  "title": "Mediterranean Quinoa Bowl",
+  "semantic_description": "A protein-rich bowl with quinoa, chickpeas, fresh vegetables, and herbs. This nutritious Mediterranean-inspired dish is easy to prepare and perfect for a quick, healthy dinner.",
+  "author": "Jamie Oliver",
+  "date": "2023-06-01",
+  "category": "Main Course",
+  "tags": ["vegetarian", "healthy", "mediterranean", "quinoa"],
+  "rating": 4.7
+}
+```
+
+### Perform semantic search
+
+Once the document has been processed by the underlying model running on the inference endpoint, you can perform semantic searches. Use the [`MATCH` function](/reference/query-languages/esql/functions-operators/search-functions.md#esql-match) (or the [`:` operator](/reference/query-languages/esql/functions-operators/operators.md#esql-match-operator) shorthand) on `semantic_text` fields to perform semantic queries:
+
+```esql
+FROM cooking_blog METADATA _score
+| WHERE semantic_description:"I need plant-based meals that aren't difficult to make"
+| SORT _score DESC
+| LIMIT 5
+```
+
+When you use `MATCH` or `:` on a `semantic_text` field, {{esql}} automatically performs a semantic search rather than a lexical search. This means the query finds documents based on semantic similarity, not just keyword matching.
+
+:::{tip}
+If you'd like to test out the semantic search workflow against a large dataset, follow the [semantic-search-tutorial](docs-content://solutions/search/semantic-search/semantic-search-semantic-text.md).
+:::
+
+### Perform hybrid search
+
+Hybrid search combines different search strategies (like lexical and semantic search) to leverage the strengths of each approach. {{esql}} provides two ways to implement hybrid search:
+
+::::{tab-set}
```

Comment:
I like the idea of tabs here - but as I said in a different comment let's remove the example with OR.
What we can have here is tabs for RRF and linear using FORK and FUSE.

Reply:
cool 👍